### DISCUSSION TITLE
So, what are these grades/ratings?

### DISCUSSION THREAD

**Discussion by YoshiElement** [KadeArchive/Kade-Engine#476] [Q&A] [answered]
I don't know if I've been looking in the wrong place or not, but I have yet to find a guide for what these ratings actually mean. I've seen many different ratings, including **FC, SDCB,** and even simply, **Clear.** I have vague ideas as to what some of these are, but what do they all mean? What are their requirements? What do the abbreviations stand for? I've also attached screenshots of all the ratings I could find.

![ss1](https://user-images.githubusercontent.com/34923812/118376786-c1204380-b58f-11eb-8428-12e0dd29f015.jpg)
![ss2](https://user-images.githubusercontent.com/34923812/118376787-c1204380-b58f-11eb-9282-e572cd0329f5.jpg)
![ss3](https://user-images.githubusercontent.com/34923812/118376791-c2ea0700-b58f-11eb-9f30-f20324e209c8.jpg)
![ss4](https://user-images.githubusercontent.com/34923812/118376792-c2ea0700-b58f-11eb-9b3f-3b33947e6ec9.jpg)
![ss5](https://user-images.githubusercontent.com/34923812/118376794-c41b3400-b58f-11eb-8d2f-a03c9c8d308a.jpg)

**Comment by puyoxyz** [comment, marked as answer]
FC = Full Combo (no combo breaks)

SDCB = Single Digit Combo Break. (9 or less combo breaks)

They are the same as Etterna's.

![image](https://user-images.githubusercontent.com/15311104/118380679-e048c080-b5e3-11eb-8da7-90c0961b9a46.png)

![image](https://user-images.githubusercontent.com/15311104/118380686-f5255400-b5e3-11eb-950f-86d9507745c5.png)

(Instead of MFC (all marvelous) and PFC (1 or more perfects), Kade Engine has MFC (all sicks) and GFC (1 or more goods))

**Reply by YoshiElement** [reply]
thank you! :)
